index.html

```diff
@@ -59,5 +57,159 @@ <h1>
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
```

Comment:
- [ ] This part of your project should be wrapped in `ul and li` tags  not  `div and paragraph`, please correct it